### PR TITLE
PP-8844 - Add timer metric for Redis calls

### PR DESCRIPTION
**Pull request by nimalank7**
Description:
- This is so we can get some useful metrics for how quickly Redis performs increment and expire values

### CODE REVIEW

File: src/main/java/uk/gov/pay/api/filter/ratelimit/RedisRateLimiter.java [hunk 1]
```diff
@@ -52,13 +57,17 @@ void checkRateOf(String accountId, RateLimiterKey key)
         }
     }
 
-    private Long updateAllowance(String key, int rateLimitInterval) {
+    private Long updateAllowance(String key, int rateLimitInterval) throws Exception {
         String derivedKey = getKeyForWindow(key, rateLimitInterval);
-        Long count = redisClientManager.getRedisConnection().sync().incr(derivedKey);
-        redisClientManager.getRedisConnection().sync().expire(derivedKey, rateLimitInterval / 1000);
+        Long count = time("redis.incr", () -> redisClientManager.getRedisConnection().sync().incr(derivedKey));
```
Comment (kbottla) [inline src/main/java/uk/gov/pay/api/filter/ratelimit/RedisRateLimiter.java:62]: Can we make the granularity of this metric clear  ? May be rename it to `redis.incr_[ns/nanoseconds]`

File: src/main/java/uk/gov/pay/api/filter/ratelimit/RedisRateLimiter.java [hunk 2]
```diff
@@ -52,13 +57,17 @@ void checkRateOf(String accountId, RateLimiterKey key)
...
+        time("redis.expire", () -> redisClientManager.getRedisConnection().sync().expire(derivedKey, rateLimitInterval / 1000));
```
Comment (kbottla) [inline src/main/java/uk/gov/pay/api/filter/ratelimit/RedisRateLimiter.java:63]: Same as above

File: src/main/java/uk/gov/pay/api/filter/ratelimit/RedisRateLimiter.java (hunk 1 above)
Comment (nimalank7) [reply src/main/java/uk/gov/pay/api/filter/ratelimit/RedisRateLimiter.java]: Done.

File: src/main/java/uk/gov/pay/api/filter/ratelimit/RedisRateLimiter.java (hunk 2 above)
Comment (nimalank7) [reply src/main/java/uk/gov/pay/api/filter/ratelimit/RedisRateLimiter.java]: Done.

**Review by kbottla — APPROVED** [review]
LGTM! Can we verify the behaviour in test/staging before it is deployed to prod.

**Pull request merged**